test/customer_test.rb

```diff
@@ -5,8 +5,8 @@
 class CustomerTest < MiniTest::Test
   attr_reader :parent, :customer
   def setup
-    sales_engine = SalesEngine.new
-    parent = CustomerRepository.new('test/support/customers.csv', sales_engine)
+    sales_engine = SalesEngine.new('test/support')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.